### PR TITLE
Updated Azure CLI commands

### PR DESCRIPTION
**Pull request by robece**
- Updated missing parameters in Azure CLI execution for pipelines and variable-groups. 
- Updated master name to main name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,8 @@ TENANT_ID=
 SUBSCRIPTION_ID=
 ACR_NAME=
 ACR_PASSWORD=
+ORGANIZATION=https://dev.azure.com/your-organization
```
Comment (fedeoliv) [inline README.md:137]: Hey @robece there is no need to add `organization` and `project` on every `az` command. if you run the following command to configure azure devops CLI: `az devops configure --defaults organization=https://dev.azure.com/your-organization project=YourProject`. This command is already on the `Getting Started` section.

File: README.md [hunk 2]
```diff
@@ -168,23 +178,33 @@ az pipelines variable-group variable create --group-id $SETT_GROUP_ID --secret t
...
+REPOSITORY_NAME=jmeter-load-test
+ORGANIZATION=https://dev.azure.com/your-organization
+PROJECT=your-project
 
 az pipelines create --name $PIPELINE_NAME_DOCKER --repository $REPOSITORY_NAME \
-    --repository-type tfsgit --branch master \
-    --yml-path pipelines/azure-pipelines.docker.yml
+                                                 --organization $ORGANIZATION \
+                                                 --project $PROJECT \
+                                                 --repository-type tfsgit --branch main \
```
Comment (fedeoliv) [inline README.md:188]: @robece loved the approach to set `main` branch as default with the `--repository-type tfsgit --branch main` command option. Thank you!

Comment (fedeoliv) [reply README.md:188]: Could you please remove the `--organization` and `--project` from the command?

**Review by fedeoliv — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (allantargino) [reply README.md:188]: I fixed it on #45. Thanks @robece

**Pull request closed without merging**